rocketpy/mathutils/function.py

```diff
@@ -39,14 +39,9 @@ def __init__(
 
         Parameters
         ----------
-        source : function, scalar, ndarray, string
-            The actual function. If type is function, it will be called for
-            evaluation. If type is int or float, it will be treated as a
-            constant function. If ndarray, its points will be used for
-            interpolation. An ndarray should be as [(x0, y0, z0), (x1, y1, z1),
-            (x2, y2, z2), ...] where x0 and y0 are inputs and z0 is output. If
-            string, imports file named by the string and treats it as csv.
-            The file is converted into ndarray and should not have headers.
+        source : function, scalar, ndarray, string, Function
+            The data source to be used for the function. See the ``set_source``
+            method's documentation for more details.
```

Comment:
@Gui-FernandesBR, I wonder if it is fair to forward users to another documentation in a basic description of constructor arguments.

I understand your idea here, to remove duplicate docs, but it can be quite frustrating when you are trying to understand how to initialize a class and you must navigate through its methods to do so.

Reply:
Same as above